concourse/common/get_review_environment

```diff
@@ -7,33 +7,42 @@ then
 fi
 
 function github {
-  curl -s -H "Authentication: token $GITHUB_ACCESS_TOKEN" "https://api.github.com/$1"
+  response=$(curl -s -H "Authentication: token $GITHUB_ACCESS_TOKEN" "https://api.github.com/$1")
+  echo "https://api.github.com/$1">&2
+  echo "$response">&2
+  echo "$response"
```

Comment:
do we want to leave all of these 'echo`es in this file?

Reply:
yea they are captured in the build logs and have been helpful for debugging